### PR TITLE
Add LoadingSpinner to homepage

### PR DESCRIPTION
**Pull request by AmyJenSandoval**
https://www.loom.com/share/9ab87a2f38b144d0a087f6ef23daff9b

### CODE REVIEW

File: src/modules/home/components/Postings.js [hunk 1]
```diff
@@ -18,7 +19,11 @@ export const Postings = () => {
 
     if (!posts || !posts.length) {
```
Comment (Leigh-K) [inline src/modules/home/components/Postings.js]: We shoulkdnt be using this logic for loading. If our app had npo posts we would get a infinite spinner. 
Add in a useState for loading
const [isLoading, setIsLoading] = useState(false)

Usually a site would have a whole other page for empty states. eg Wow it looks quiet today. That type of shit you see.

File: src/modules/add-post/components/AddPost.js [hunk 2]
```diff
@@ -30,7 +33,13 @@ export const AddPost = () => {
                     }
                 }
             )
-            .catch((err) => console.log(err))
+
+            if (response) {
+                navigate(myPostsPath)
```
Comment (Leigh-K) [inline src/modules/add-post/components/AddPost.js:38]: good idea

File: src/modules/home/components/Postings.js (hunk 1 above)
Comment (AmyJenSandoval) [reply src/modules/home/components/Postings.js]: done

File: src/modules/home/components/Postings.js [hunk 3]
```diff
@@ -2,38 +2,54 @@ import React, { useEffect, useState } from 'react'
...
-    }
+        const getPosts = async () => {
+            setIsLoading(true)
+            try {
+                await axios
+                    .get(
+                        // eslint-disable-next-line no-undef
+                        `${process.env.REACT_APP_SERVICE_ENDPOINT}/post/getPosts`
+                    )
+                    .then((res) => {
```
Comment (Leigh-K) [inline src/modules/home/components/Postings.js]: remoe the .then it doesnt do anything now that we are using await
await axios
                    .get(
                        // eslint-disable-next-line no-undef
                        `${process.env.REACT_APP_SERVICE_ENDPOINT}/post/getPosts`
                    )
setPosts(res.data.posts)
setIsLoading(false)
  } catch (error) {
                console.log(error)
            }

Comment (AmyJenSandoval) [reply src/modules/home/components/Postings.js]: done, but had undefined error, so had to define response

**Review by Leigh-K — APPROVED** [review]

**Pull request merged**